src/plugins/recordTypes/media/forms/default.jsx

```diff
@@ -0,0 +1,92 @@
+const template = (configContext) => {
+  const {
+    React,
+  } = configContext.lib;
+
+  const {
+    Col,
+    Cols,
+    Panel,
+  } = configContext.layoutComponents;
+
+  const {
+    Field,
+    Subrecord,
+  } = configContext.recordComponents;
+
+  const {
+    extensions,
+  } = configContext.config;
+
+  return (
+    <Field name="document">
+      <Panel name="media" collapsible>
+        <Panel name="file" collapsible>
+          <Cols>
+            <Col>
+              <Field name="identificationNumber" />
+              <Field name="title" />
+              <Field name="handwritten" subpath="ns2:media_botgarden" />
+            </Col>
+
+            <Col>
+              <Field name="scientificTaxonomy" subpath="ns2:media_botgarden" />
+              <Field name="imageNumber" subpath="ns2:media_botgarden" />
+              <Field name="postToPublic" subpath="ns2:media_botgarden" />
+            </Col>
+          </Cols>
+          <Subrecord name="blob" showDetachButton />
+        </Panel>
+
```

Comment:
In 5.0 let's always keep the file info panel the same as it is in core. Put the fields above it outside of the file info panel, but in the media handling info panel, as the identification number and title fields are in core.

Also let's reorder the fields so the tab order is maintained:

```
identificationNumber     imageNumber
scientificTaxonomy       handwritten
title                    postToPublic
```